cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -64,6 +64,15 @@
 
 @hook('upgrade-charm')
 def upgrade_charm():
+    # migrate to new flags
+    if is_state('kubernetes-worker.restarted-for-cloud'):
+        remove_state('kubernetes-worker.restarted-for-cloud')
+        set_state('kubernetes-worker.cloud.ready')
+    if is_state('kubernetes-worker.cloud-request-sent'):
+        # minor change, just for consistency
+        remove_state('kubernetes-worker.cloud-request-sent')
+        set_state('kubernetes-worker.cloud.request-sent')
```

Comment:
Looks like `kubernetes-worker.cloud-request-sent` is still referenced in a couple other places as well. Can you update those?